Overview.src.html

```diff
@@ -3944,11 +4021,22 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   <span title=concept-request-body>body</span> is non-null, run these substeps:
 
   <ol>
+   <li><p>Let <var>stream</var> be an <span title=concept-empty-ReadableStream>empty</span>
+   <code title=concept-ReadableStream>ReadableStream</code> object.
+
    <li><p>Set <var>input</var>'s <span title=concept-Request-request>request</span>'s
-   <span title=concept-request-body>body</span> to an empty byte stream.
+   <span title=concept-request-body>body</span> to a new <span title=concept-body>body</span> whose
+   <span title=concept-body-stream>stream</span> is <var>stream</var>.
 
-   <li><p>Set <var>input</var>'s
-   <span title=concept-Request-disturbed-flag>disturbed flag</span>.
+   <li>
```

Comment:
The operations are just to make request disturbed and locked. See https://github.com/whatwg/fetch/commit/e7d718ac577b12f190e77370db42b68745d13eac and https://github.com/yutakahirano/fetch-with-streams/issues/60.

I think using pipeTo is the right way, but it's not distinguishable at this moment because the Request class doesn't have `body` property (yet).